### DISCUSSION TITLE
Is there a way to change the theme without reboot?

### DISCUSSION THREAD

**Discussion by nordeep** [HASwitchPlate/openHASP#167] [Q&A] [answered]
Is there a way to change the theme without reboot? This will be useful to change light/dark theme on sunrise/sunset!
For now it's possible by:
Publish - {"theme":4} -> hasp/plate_01/config/hasp
Publish - reboot -> hasp/plate_01/command

**Comment by fvanroie** [comment, marked as answer]
It's not possible at this time, but it makes for an interesting use case. I'll look into it and see what can be done.

**Reply by fvanroie** [reply]
On-the-fly switching between dark/light mode is possible within the same theme. The objects update immediately and apply the updated theme styles. Switching to another theme however doesn't affect existing objects, but only new objects...

So I think mode switching should be possible without reboot, whereas theme switching would still require a reboot or complete reload to apply.

**Comment by fvanroie** [comment]
There is an experimental `theme <themeid>` command in 0.6.1-dev that can be tested.
As noted above, it only works well for switching between dark/light mode within the same theme.
